examples/credential_issuer_metadata_sd_jwt_long.json

```diff
@@ -0,0 +1,133 @@
+{
+  "credential_issuer": "https://credential-issuer.example.com",
+  "authorization_servers": [
+    "https://server.example.com"
+  ],
+  "credential_endpoint": "https://credential-issuer.example.com",
```

Comment:
Should probably have a path? e.g. "/credential" (same problem exists in the signed version too)